test/string_test.rb

```diff
@@ -85,6 +87,20 @@
     end
   end
 
+  describe '#classified' do
```

Comment:
@choallin You introduced unit tests for `#classified?`, but not for the original issue, which was related to `#classify`.